### PR TITLE
Fix access_token to be set on Redis instead of django cache, add peri…

### PR DESCRIPTION
**Pull request by norkans7**
…odic task to JC type package

### CODE REVIEW

**Comment by nicpottier** [comment]
Need to remove the task from our settings no?

**Comment by nicpottier** [comment]
coverage

File: temba/channels/types/jiochat/tasks.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from __future__ import print_function, unicode_literals
+
+from celery.task import task
+from temba.channels.models import Channel
+from temba.utils.jiochat import JiochatClient
+
+
+@task(track_started=True, name='refresh_jiochat_access_tokens')
+def refresh_jiochat_access_tokens():  # pragma: needs cover
```
Comment (nicpottier) [inline temba/channels/types/jiochat/tasks.py]: Errr.. why?

Comment (norkans7) [reply temba/channels/types/jiochat/tasks.py]: I am still trying to figure out why it is not covered on travis

**Pull request merged**